### ISSUE TITLE
Writing to file fails due to encoding

### ISSUE THREAD

**Issue by latman-oliveai** [open-olive/loop-development-kit#157]
```
 const testEncodedString = await network.encode('Hi');
    await filesystem.writeFile(
      '/Users/michaellatman/Desktop/test.txt',
      testEncodedString,
      WriteOperation.overwrite,
      54
    );
```

Results in this log:
```
"level":"info","system":"authenticated application","system":"secureloops","loopID":"8d2350cc-c138-423a-830a-28b7b1924ddc","console":"log","time":"2021-05-06T13:00:00-04:00","message":"TypeError: could not convert function call parameter 72,105 to []uint8"}

```

**Comment by cshields-oliveai** [comment]
@latman-oliveai - Thanks for reporting this. Luckily, we believe this one has been resolved and is currently slated to go out with the next LDK release.

Here is the PR for your reference: https://github.com/open-olive/loop-development-kit/pull/128 (note: there was also a desktop app change that this LDK PR will coincide with. The next release will contain both)

**Comment by CandiceCLB** [comment]
This issue has been resolved with the latest Olive Helps (V 0.17.3) and LDK (V 3.0.0.beta.3) release. Please update to these versions and ifyou find you are still having this issues, please feel free to re-open the ticket.
